css/main.css

```diff
@@ -1,3 +1,663 @@
+/*![Juice Bar homepage](./designs/Juice%20bar.png)
+
+Live demo here: https://www.wix.com/website-template/view/html/2962  
+### Fonts
+**Headings and subheadings:** Questrial - Regular 400  
+https://fonts.google.com/specimen/Questrial  
+**Buttons:** Raleway - Regular 400  
+https://fonts.google.com/specimen/Raleway  
+**Body:** Roboto - Light 300  
+https://fonts.google.com/specimen/Roboto  
+
+
 /* Add your CSS here */
 
-/* Dont' forget to link this file to your HTML in the <head> */
+:root {
+  --black: #000;
+  --white: #fff;
+  --lylac: #f5e8ff;
+  --light-green: #e6fac0;
+  --medium-green: #5e7d1f;
+  --dark-green: #394b2a;
+}
+
+body {
+  font-family: "Roboto" sans-serif;
+  font-size: 15px;
+  margin: 0;
+  line-height: 1.38; 
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  text-align: center; 
+}
+
+h1,
+h2,
+h3 {
+  font-family: "Questrial";
+  font-weight: 300;
+  margin: 0;
+}
+
+h2 {
+  font-size: 32px;
+}
+
+h3 {
+  font-size: 22px;
+}
+
+p {
+  font-family:"HelveticaNeueW01-45Ligh";
+  font-size: 15px;
+  font-weight: 70;
+  margin: 0.5rem 2.2rem;
+  line-height: 1.7;
+  letter-spacing:0.2px;
+}
+
+/* Header */
+
+button {
+  font-family: "Raleway";
+  width: 170px;
+  height: 48px;
+  font-size: 15px;
+  border-radius: 0;
+  background-color: var(--black);
+  color: var(--white);
+}
+
+.header .order-btn {
+  background-color: var(--black);
+  width: 100%;
+  color: var(--white);
+}
+
+.login {
+  display: none;
+}
+
+.header-bottom {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  text-align: left;
+  padding: 1rem 1.3rem;
+}	
+
+.bar {
```

Comment:
it's more common to use an svg icon for things like the hamburger icon. Check out [font awesome](https://fontawesome.com/v5.15/icons)